configs/js.json

```
@@ -0,0 +1,14 @@
+{
+  "defaults": {
+    "files": [
+      ".github/workflows/automerge.yml",
+      ".github/workflows/js-test-and-release.yml"
+    ]
+  },
+  "repositories": [
+    { "target": "ipfs/js-blockstore-core" },
+    { "target": "libp2p/js-libp2p-crypto" },
+    { "target": "libp2p/js-libp2p-interfaces" },
+    { "target": "libp2p/js-libp2p-tcp" }
```

```suggestion
    { "target": "libp2p/js-libp2p-tcp" },
    { "target": "alanshaw/it-pipe" },
    { "target": "alanshaw/it-goodbye" }
```

vasco-santos/is-loopback-addr should also be in here but it's primary branch is called `main` and not `master` so I guess #262 needs merging first?

---

Note that these repos will need to grant permissions to the web3-bot.

---

No, we don't have to wait for #262 because `$default-branch` string is already replaced in our templated with a name of the default branch. #262 only generalises the templating approach.

As for more repos, how about we start with the original 4 to make sure the setup works as expected first? @achingbrain Could you check if `web3-bot` is allowed to write to https://github.com/ipfs/js-blockstore-core/pull/14? I don't have access to settings in that repo.

---

Aha, gotcha.

I've added `web3-bot` to blockstore-core and the stewards team, @alanshaw is doing his various it-* repos.

Happy to merge this without the extra repos - since the bot is gaining perms we can add them later.

---

Is someone monitoring the `web3-bot` inbox for repo invites?  They have to be manually approved by `web3-bot` if you're not adding it to a repo from an org it's already a member of.